test/ecdhtest.c

```diff
@@ -462,6 +461,105 @@ static int ecdh_kat(BIO *out, const ecdh_kat_t *kat)
     return rv;
 }
 
+#include "ecdhtest_cavs.h"
+
+static int ecdh_cavs_kat(BIO *out, const ecdh_cavs_kat_t *kat)
```

Comment:
Can you add some small comment about what this function tests and what the return value is?